recipes/KsponSpeech/ASR/transformer/README.md

```diff
@@ -25,7 +25,9 @@ python train.py hparams/conformer_medium.yaml
 # Results
 | Release  |   hyperparams file    | eval clean WER | eval other WER | eval clean CER | eval other CER |                                   HuggingFace link                                   |                                               Model link                                                |    GPUs     |  Training time  |
 | :------: | :-------------------: | :------------: | :------------: | :------------: | :------------: | :----------------------------------------------------------------------------------: | :-----------------------------------------------------------------------------------------------------: | :---------: | :-------------: |
-| 01-23-23 | conformer_medium.yaml |     20.47%     |     25.18%     |     7.33%      |     7.99%      | [HuggingFace](https://huggingface.co/speechbrain/asr-conformer-transformerlm-ksponspeech) | [DropBox](https://www.dropbox.com/sh/uibokbz83o8ybv3/AACtO5U7mUbu_XhtcoOphAjza?dl=0) | 6xA100 80GB | 2 days 13 hours |
+| 04-16-24 | conformer_medium.yaml |     20.15%     |     24.75%     |     7.40%      |     7.96%      | [HuggingFace](https://huggingface.co/ddwkim/asr-conformer-transformerlm-ksponspeech) | [DropBox](https://www.dropbox.com/sh/uibokbz83o8ybv3/AACtO5U7mUbu_XhtcoOphAjza?dl=0) | 2xA100 40GB | 14 hours 7 mins |
```

Comment:
I am a bit curious but do you know why we went from 2days and 13 hours of training using 6x A100 to "only" 14 hours and 7 mins of training using 2x A100 40GB?

Reply:
I assumed there may have been optimizations in the speechbrain train pipeline and higher torch versions, which I could not enjoy in mid 2022, the year I trained the model last time. I will share the train logs soon, and share findings if there is anything found during inspection of code changes between now and then.